README.md

```diff
@@ -1 +1,36 @@
-## Battleship
+
+# Battleship game
+
+
+##### Credits: Eli Sachs & Kevin Ta
+###### Turing School of Software & Design
+
+
+### <font color="gree">Introduction:</font>
+Welcome players! This game of Battleship was created using Ruby code and is run directly in your terminal command line.
+
+Run `battleship_runner.rb` located in root of the `battleship` folder to begin the game.
```

Comment:
Can we change this to 'run `**ruby** battleship_runner.rb`?

Reply:
sure! I'll switch it now